test/MicrosoftTeams.spec.ts

```diff
@@ -169,20 +181,63 @@ describe("MicrosoftTeams", () => {
     let getContextMessage = findMessageByFunc("getContext");
     expect(getContextMessage).not.toBeNull();
 
-    processMessage({
-      origin: "https://some-malicious-site.com/",
-      source: microsoftTeams._window.parent,
-      data: {
-        id: getContextMessage.id,
-        args: [
-          {
-            groupId: "someMaliciousValue"
-          }
-        ]
-      } as MessageResponse
-    } as MessageEvent);
+    unSupportedDomains.forEach(unSupportedDomain => {
+      processMessage({
+        origin: unSupportedDomain,
+        source: microsoftTeams._window.parent,
+        data: {
+          id: getContextMessage.id,
+          args: [
+            {
+              groupId: "someMaliciousValue"
+            }
+          ]
+        } as MessageResponse
+      } as MessageEvent);
+
+      expect(callbackCalled).toBe(false);
```

Comment:
Similarly here it checks that the callback was never called (which is correct), but should it fail and a callback does end up getting called, we don't keep enough information to identify which one was missed.